doc/sphinx/source/esmvaldiag/fixing_data.rst

```diff
@@ -0,0 +1,221 @@
+.. fixing_data:
```

Comment:
Please rename this file to `.inc`, for consistency.

Reply:
Why we end those at .inc? This confuses IDEs quite a bit

Reply:
I know, but after yesterday's discussion we decided to keep `.inc` (can't remember why, though).

Reply:
I will fix this when I finish the text, so I benefit from the IDE preview as much as possible

Reply:
At some point I had to rename all .rst documents that are included in another .rst document (using the `.. include:: ` directive) to .inc, because sphinx was not only including but also trying to build those as separate documents and complaining about that. Maybe this has been fixed in sphinx, if your documentation builds fine without warnings, I think it should be ok to go back to .rst.